docs/tutorials/03_emulation_sensitivity.ipynb

```diff
@@ -0,0 +1,516 @@
+    "    peak_pressure = y[:ncomp, :].max()\n",
+    "    Y.append(peak_pressure)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Emulate & Validate\n",
+    "\n",
+    "Here we setup Autoemulate. \n",
+    "\n",
+    "The models currently available for performing cross-validation are as follows: \n",
+    "- sop: SecondOrderPolynomial\n",
+    "- rbf: RadialBasisFunctions\n",
+    "- rf: RandomForest\n",
+    "- gb: GradientBoosting\n",
+    "- lgbm: LightGBM\n",
+    "- svm: SupportVectorMachines\n",
+    "- gp: GaussianProcess\n",
+    "- cnp: ConditionalNeuralProcess\n",
+    "- acnp: AttentiveConditionalNeuralProcess\n",
+    "- gpmt: GaussianProcessMT\n",
+    "- gps: GaussianProcessSklearn\n",
+    "- nns: NeuralNetSk\n",
+    "\n",
+    "We have selected four of these models for cross-validation, feel free to explore more!  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "em = AutoEmulate()\n",
+    "parameter_names = list(parameters_range.keys())\n",
+    "em.setup(sample_df[parameter_names], Y, models = ['gp','cnp'])\n",
+    "best_model = em.compare()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "em.summarise_cv()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "em.plot_eval(model=best_model)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "em.evaluate(best_model)  \n",
+    "best_emulator = em.refit(best_model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Sensitivity Analysis\n",
+    "\n",
+    "1. Define the problem by creating a dictionary which contains the names and the boundaries of the parameters \n",
+    "2. contribution of each parameter , importance of each parameter \n",
+    "    - S₁: First-order sensitivity index.\n",
+    "    - S₂: Second-order sensitivity index.\n",
+    "    - Sₜ: Total sensitivity index."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Extract parameter names and bounds from the dictionary\n",
+    "parameter_names = list(parameters_range.keys())\n",
+    "parameter_bounds = list(parameters_range.values())\n",
+    "\n",
+    "# Define the problem dictionary for Sobol sensitivity analysis\n",
+    "problem = {\n",
+    "    'num_vars': len(parameter_names),\n",
+    "    'names': parameter_names,\n",
+    "    'bounds': parameter_bounds\n",
+    "}\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "em.sensitivity_analysis(problem=problem)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
I'm a bit confused by everything that follows from here onwards. The above cell demonstrates how to do sensitivity analysis with AutoEmulate but now it seems that we've reverted to using the SALib package to do some further custom analysis? I'd remove the rest of this section and if there is anything here that we think that autoemulate should capture we can add it to an issue to tackle separately.